unicoremc/models.py

```diff
@@ -63,17 +63,26 @@ class Meta:
 
 
 class AppType(models.Model):
+    UNICORE_CMS = 'unicore-cms'
+    SPRINGBOARD = 'springboard'
+    PROJECT_TYPES = (
+        (UNICORE_CMS, 'unicore-cms'),
+        (SPRINGBOARD, 'springboard'),
+    )
+
     name = models.CharField(max_length=256, blank=True, null=True)
     title = models.TextField(blank=True, null=True)
+    project_type = models.CharField(
+        choices=PROJECT_TYPES, max_length=256, default=UNICORE_CMS)
```

Comment:
`default=SPRINGBOARD` instead? But keeping it so that the migration stores `UNICORE_CMS` for existing apps